features/session/session.ts

```
@@ -1,252 +1,796 @@
+  async initData(): Promise<void> {
+    if (!this._request) {
+      this._log("No request available for session initialization", undefined, "warn");
+      return;
+    }
+
+    try {
+      const sessionType = globalThis.serverConfig?.session?.type;
+
+      switch (sessionType) {
+        case "cookie":
+          const sessionData = this._request.webtoken.session();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)